custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -169,7 +169,12 @@ function DownloadController($scope, $rootScope, $location, locationHierarchy, lo
         ];
         vm.selectedBeneficiaryCategory = 'pw_lw_children';
     }
-
+    vm.THRreportType = [
+        {id: 'consolidated', name: 'Consolidated'},
+        {id: 'beneficiary_wise', name: 'Beneficiary wise'},
+        {id: 'days_beneficiary_wise', name: 'Days & Beneficiary wise'},
+    ];
+    vm.selectTHRreportType = 'consolidated';
```

Comment:
not blocking: but good to use `selectedTHRreportType` instead of `selectTHRreportType `